SU-SOE/stanford_soe_helper
# 7.x 2.x soe 3153 banner bar

# READY FOR Visual REVIEW


# Summary
- For ticket: https://stanfordits.atlassian.net/browse/SOE-3153
- This adds the colored bar to the caption of the top banner.

As I am out next week, @minorwm will you please put this up for Kerri to review. If it passes, will you please merge? Thank you! - Caryl

# Needed By (Date)
- Sprint end

# Criticality
- For Giving: Client is eagerly awaiting this.

# Steps to Test
- Switch to this branch `git checkout 7.x-2.x-SOE-3153-banner-bar`
- Clear cache 
- Create or update three Stanford Page nodes, and for each
  - if needed, add a top banner image
  - add a caption to the top banner
  - add a different color (turquoise, orange, pink) 
- Save
- Verify you see the proper color bar on the top banner for each node.

![screen shot 2018-06-27 at 17 17 22](https://user-images.githubusercontent.com/284440/42006322-07469846-7a2e-11e8-8f56-6fb700c516f7.png)


# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
Dev: https://stanfordits.atlassian.net/browse/SOE-3153
PR: https://stanfordits.atlassian.net/browse/SOE-3207
## Related PRs


## More Information
Responsive will be addressed here: https://stanfordits.atlassian.net/browse/SOE-3154 

## Folks to notify
@kerri-augenstein 


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
